terraform/cp4ba/main.tf

```diff
@@ -16,8 +16,11 @@ resource "null_resource" "mkdir_kubeconfig_dir" {
   }
 }
 
-module "cluster" {
-  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/main/modules/roks"
+module "create_cluster" {
+//  source = "github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/terraform-0.13/modules/roks"
+//  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/terraform-0.13/modules/roks"
+  source = "../../../joel_cloud_pak_terraform_issue220/modules/roks"
```

Comment:
Why is this using the roks on your branch?  You should be using   `source = "github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/roks?ref=terraform-0.13"`
 for now.

Reply:
I fixed it using this branch:  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/terraform-0.13/modules/roks"